api/src/main/java/jakarta/resource/spi/ManagedConnection.java

```diff
@@ -209,7 +209,6 @@ void removeConnectionEventListener(
    *  @return     LocalTransaction instance
    *
    *  @throws     ResourceException     generic exception if operation fails
-   *  @throws     NotSupportedException if the operation is not supported
```

Comment:
Was this removed by mistake?

Reply:
No it's not in the method signature so javadoc complains

Reply:
It isn't valid to remove this.  See my reply https://github.com/eclipse-ee4j/jca-api/pull/106#discussion_r471574936 for details.

Reply:
I reinstated the throws and added the import